.github/workflows/build.yml

```
@@ -0,0 +1,55 @@
+        env:
+          REPOSITORY: '${{ github.repository }}'
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{env.REPOSITORY_LC}}
+
+      - name: Login to GHCR
+        if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'push-image')
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)